build/Common.nonprod.props

```diff
@@ -17,17 +17,27 @@
     </Content>
   </ItemGroup>
 
-  <PropertyGroup Label="Package versions used in by test projects">
+  <PropertyGroup Label="Package versions used by test and example projects">
```

Comment:
Makes me wonder if we actually want a separate .props file for examples... `Common.examples.props`.

Reply:
It might introduce duplication.